test/config.ts

```diff
@@ -43,7 +43,7 @@ if (process.env.NETWORK_NAME === 'testing') {
   Object.assign(configBase, {
     web3ProviderUri: 'http://contracts.nevermined.localnet',
     marketplaceUri: 'http://marketplace.nevermined.localnet',
-    neverminedNodeUri: 'http://node.nevermined.localnet',
+    neverminedNodeUri: 'http://localhost:8030',
```

Comment:
In the testint-node.yml you are specifying `NEVERMINED_NODE_URI = http://localhost:8030` so probably here should be `process.env.NEVERMINED_NODE_URI || 'http://node.nevermined.localnet' assuming we use NVM Tools by default